File: src/main/java/eu/mshade/enderframe/item/metadata/EnchantmentsItemStackMetadata.java
package eu.mshade.enderframe.item.metadata;

import eu.mshade.enderframe.item.Enchantment;
import eu.mshade.enderframe.metadata.MetadataKeyValue;
import eu.mshade.enderframe.item.ItemStackMetadataKey;

import java.util.List;

public class EnchantmentsItemStackMetadata extends MetadataKeyValue<List<Enchantment>> {

    public EnchantmentsItemStackMetadata(List<Enchantment> metadataValue) {
        super(ItemStackMetadataKey.ENCHANTMENTS, metadataValue);
    }

}

File: src/main/java/eu/mshade/enderframe/item/metadata/EnchantmentsItemStackMetadata.kt
package eu.mshade.enderframe.item.metadata

import eu.mshade.enderframe.item.Enchantment
import eu.mshade.enderframe.item.ItemStackMetadataKey
import eu.mshade.enderframe.metadata.MetadataKeyValue

class EnchantmentsItemStackMetadata(metadataValue: List<Enchantment>) :
    MetadataKeyValue<List<Enchantment>>(ItemStackMetadataKey.ENCHANTMENTS, metadataValue)
